apps/coscrad-frontend/src/components/resources/shared/resource-nav-link.tsx

```diff
@@ -1,16 +1,24 @@
 import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
-import { Box, IconButton } from '@mui/material';
+import { Box, IconButton, SxProps, Theme } from '@mui/material';
 import { Link } from 'react-router-dom';
 
 interface ResourceNavLinkProps {
     internalLink: string;
+    /**
+     * Note: iconSx is a temporary workaround, we may need more than one
+     * component to display the resource navigation link depending on context
+     */
+    iconSx: SxProps<Theme>;
```

Comment:
The trouble here is that we have responsive size in the detail presenters, but then in the marker popup in spatial features we need a consistent size because the popup remains the same size regardless of screen size.

Reply:
Is there a way to opt out of using the generic helper for spatial features only? If not, I need to make it so. Use the "escape hatch" for this one.